wikichange/src/content/graph.js

```diff
@@ -0,0 +1,88 @@
+import Chart from 'chart.js/auto';
+
+const CHART_COLORS = {
+    red: 'rgb(255, 99, 132)',
+    orange: 'rgb(255, 159, 64)',
+    yellow: 'rgb(255, 205, 86)',
+    green: 'rgb(75, 192, 192)',
+    blue: 'rgb(54, 162, 235)',
+    purple: 'rgb(153, 102, 255)',
+    grey: 'rgb(201, 203, 207)'
+};
+
+/* Creates the div for the graph overlay. TODO: create the graph and render it here */
+let renderGraphOverlay = () => {
+    let graphContainer = document.createElement('div');
+    let canvas = document.createElement('canvas');
+    canvas.style.maxHeight = '150px';
+    canvas.id = 'myChart';
+    graphContainer.appendChild(canvas);
+
+    let siteSub = document.getElementById('siteSub');
+    siteSub.append(graphContainer);
+}
+
+renderGraphOverlay();
+
+const labels = [
+    'January',
+    'February',
+    'March',
+    'April',
+    'May',
+    'June',
+];
```

Comment:
Have you figured out the way to add a lot of data and only have the x-axis show some labels and not all (so it is still readable)? What if we only want the labels to show only the months but the data itself to be per day, so we have a more detailed graph?

Reply:
This may be what you are looking for https://www.chartjs.org/docs/latest/configuration/decimation.html, I have not had time to look into it yet.

Reply:
Looks legit. We can merge this and I will replace the data with a real one in my pr. Will just do an ugly version that shows all the x values first and can implement this plugin later.

Reply:
looks good! yes, this is what I was looking at too! https://www.chartjs.org/docs/latest/configuration/decimation.html
https://www.chartjs.org/docs/latest/configuration/interactions.html